### PR TITLE
Make travis worker path if it does not exist.

### PR DESCRIPTION
**Pull request by ericroberts**

### CODE REVIEW

File: app/travis_job.rb [hunk 1]
```diff
@@ -4,7 +4,11 @@
 
 module Tint
 	class TravisJob
-		QUEUE_DIR = Pathname.new(ENV["TRAVIS_WORKER_BASE_DIR"].to_s).realpath
+		QUEUE_DIR = begin
```
Comment (singpolyma) [inline app/travis_job.rb]: maybe move ensure_path up somewhere and use that?

File: app/travis_job.rb [hunk 2]
```diff
@@ -4,7 +4,11 @@
...
+			path = Pathname.new(ENV["TRAVIS_WORKER_BASE_DIR"].to_s)
+			path.mkpath
```
Comment (singpolyma) [inline app/travis_job.rb]: Should only mkpath if environment variable was set

**Comment by ericroberts** [comment]
@singpolyma ready for re-review

File: app/site.rb [hunk 3]
```diff
@@ -178,10 +179,9 @@ def maybe_commit(git, message, user)
 		end
 
 		def ensure_path(key, env)
-			@options[key] ||= Pathname.new(ENV.fetch(env)).
-			                  realpath.join(@options[:site_id].to_s)
-			@options[key].mkpath
-			@options[key]
+			@options[key] ||= PathHelpers.ensure_path(
```
Comment (singpolyma) [inline app/site.rb]: path should be ensured even if not from the env var

**Comment by ericroberts** [comment]
@singpolyma ready again

**Pull request merged**